base/file.jl

```diff
@@ -658,6 +658,7 @@ end
 Change the permissions mode of `path` to `mode`. Only integer `mode`s (e.g. `0o777`) are
 currently supported. If `recursive=true` and the path is a directory all permissions in
 that directory will be recursively changed.
+Returns `path`.
```

Comment:
Returns -> Return I think.

Reply:
I think Returns, as in "This function returns `path`"

Overly complicated explanation for why:
It is a sentence incorporating ellipsis. which is short for "This function returns `path`"
Returns is the verb in 3rd person present tense (I had to look that up).

Reply:
The imperative mode guideline was introduced in #15136, among several other guidelines, by @nalimilan. However, I can't seem to find discussion of that particular aspect in the PR. Was it discussed elsewhere, e.g. on discourse or slack?

(FWIW, I do agree with @KristofferC that indicative mode sounds more natural, as neither the author nor the reader of the documentation will be the actor performing the returning action... but I may be missing something.)

Reply:
I don't remember where that rule comes from, but I don't think I invented it. It's a know problem that our docs are inconsistent, and I don't really care about which convention is chosen as long as one exists.

Reply:
I do know of that [convention/recommendation for git commits](https://chris.beams.io/posts/git-commit/), but not for function documentation. I do think it makes more sense in the former context than the latter, but don't meant to discuss this here. Should I open an issue for that, or is everyone else ok with the current convention?

Reply:
Note that the cited rule applies to the one-line sentence in the following sense:
>  Include a single one-line sentence describing what the function does or what the object represents after the simplified signature block. If needed, provide more details in a second paragraph, after a blank line

So no need to discuss the rule here, because it doesn't apply.